src/catalog/src/kvbackend/manager.rs

```diff
@@ -555,6 +554,7 @@ pub(super) struct SystemCatalog {
     // system_schema_provider for default catalog
     pub(super) information_schema_provider: Arc<InformationSchemaProvider>,
     pub(super) pg_catalog_provider: Arc<PGCatalogProvider>,
+    pub(super) numbers_table_provider: Arc<NumbersTableProvider>,
```

Comment:
nit: The `NumbersTableProvider` is already cloneable.
```suggestion
    pub(super) numbers_table_provider: NumbersTableProvider,
```